src/SavedSearches/Properties/CreatedByQueryString.php

```diff
@@ -25,6 +45,19 @@ public function __construct(UserId $userId)
      */
     private function generateQuery(): string
     {
-        return 'createdby:' . $this->userId->toNative();
+        switch ($this->createdByQueryMode->toNative()) {
```

Comment:
Dit was nog iets properder geweest met een factory en 1 klasse per mode ipv 1 CreatedByQueryString. De factory zou dan op basis van de geconfigureerde methode en de culturefeed user, een instance van de juiste CreatedByQueryString klasse kunnen aanmaken. FixedSavedSearchRepository zou in principe ook als factory kunnen fungeren, want die heeft de query mode al, en delegeert nu eigenlijk quasi alles al naar CreatedByQueryString.

Reply:
Variant op wat ik hierboven voorstelde: ipv een aparte CreatedByQueryString klasse per mode, is de huidige klasse behouden maar in de factory afh. van de mode 1 of meerdere strings als parameters aan de constructor van CreatedByQueryString meegeven (echt een collectie van strings, dus geen specifieker getypete objecten). In generateQuery() gebruik je dan juiste syntax voor 1 of meerdere waarden.

Mijn voorkeur gaat naar deze oplossing.